src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -52,31 +52,30 @@ public void storeMessage(Message message) {
    * @return a list of messages posted by the user, or empty list if user has never posted a
    *     message. List is sorted by time descending.
    */
-  public List<Message> getMessages(String user) {
```

Comment:
Why not just keep both getMessages(String user) and getAllMessages()? Then we can have the methods for both getting a single user's messages and all messages from all users.